elements/lazy-iframe/lazy-iframe.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { registerReactElement } from 'bulbs-elements/register';
+import { loadOnDemand } from 'bulbs-elements/util';
+
+registerReactElement(
+  'lazy-iframe',
+  loadOnDemand((props) => (
+    <iframe
+      {...props}
+      style={{
+        width: '100%',
+        height: '100%',
+      }}
+    />
+  ))
+);
```

Comment:
Looking to merge this.

This strategy could also be used with the `bundle-loader` to lazily load implementation for elements on demand: https://gist.github.com/collin/9b0a6ae2093ed70ad4d46b94a3fc3c3b